src/TerminalFighter/Title/Title.h

```diff
@@ -0,0 +1,17 @@
+#include <string>
+#include <SDL_ttf.h>
+
+class Title {
+    std::string title_str;
```

Comment:
The order of members in this class is super weird. Should match the rest of the project.